specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.pricesheets.json

```diff
@@ -0,0 +1,284 @@
+                "type": "string"
+              },
+              "OData-EntityId": {
+                "description": "The operation entity Id GUID.",
+                "type": "string"
+              }
+            }
+          },
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/DownloadUrl"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Error": {
+      "description": "Error details for transfer execution.",
+      "type": "object",
+      "properties": {
+        "errorCode": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Error code."
+        },
+        "errorMessage": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Error message."
+        }
+      }
+    },
+    "ErrorDetails": {
+      "description": "The details of the error.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ErrorResponse": {
+      "description": "Error response indicates that the service is not able to process the incoming request. The reason is provided in the error message. \n\nSome Error responses: \n\n * 429 TooManyRequests - Request is throttled. Retry after waiting for the time specified in the \"x-ms-ratelimit-microsoft.consumption-retry-after\" header. \n\n * 503 ServiceUnavailable - Service is temporarily unavailable. Retry after waiting for the time specified in the \"Retry-After\" header.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The details of the error.",
+          "$ref": "#/definitions/ErrorDetails"
+        }
+      }
+    },
+    "DownloadUrl": {
+      "description": "A secure URL that can be used to download a an entity until the URL expires.",
+      "type": "object",
+      "properties": {
+        "expiryTime": {
+          "description": "The time in UTC when the download URL will expire.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "url": {
+          "description": "The URL to the ZIP file. This Zip file will consists of multiple CSV files",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ProxyResource": {
```

Comment:
I would suggest instead, removing your definition of proxy resource here, inheriting ProxyResource or EntityResource from 

          {
            "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource" and adding the extra etag property that I think is not included in ProxyResource.